OloEngine/src/Platform/Linux/LinuxPlatformMemory.cpp

```
@@ -0,0 +1,132 @@
+        outConstants.OsAllocationGranularity = 65536;
+
+        outConstants.TotalPhysical = ReadMemInfoValue("MemTotal");
+        const long pages = sysconf(_SC_PHYS_PAGES);
+        if (outConstants.TotalPhysical == 0 && pageSize > 0 && pages > 0)
+        {
+            outConstants.TotalPhysical = static_cast<u64>(pageSize) * static_cast<u64>(pages);
+        }
+        // Virtual memory limit isn't directly queryable; leave as 0.
+    }
+
+    void* AllocateFromOS(sizet size)
```

Cppcheck (2.20.0)

[style] The function 'AllocateFromOS' is never used.

(unusedFunction)